### DISCUSSION TITLE
Sync settings²

### DISCUSSION THREAD

**Discussion by stanthewizzard** [vrtmrz/obsidian-livesync#286] [Q&A]
Hello

Is there a way with latest version to sync settings  & plugins between obsidians ?

Thanks for help

**Comment by dosoft** [comment]
Customization sync appears to be the solution here. However, it doesn't consistently deliver the desired results. For instance, take the Icon Folder plugin - it stores iconsets within the plugins folder as a subfolders, and unfortunately, the customization feature doesn't seem to address this issue effectively. On the bright side, it generally works quite well when dealing with the JSON config files for the plugins

**Reply by stanthewizzard** [reply]
Thanks I'll try

**Comment by dosoft** [comment]
Btw, @vrtmrz, do you think it's possible to support this? I mean, in case the plugin stores data outside of JSON files, like the Icon Folder plugin (https://obsidian.md/plugins?id=obsidian-icon-folder).

**Reply by vrtmrz** [reply]
In customisation sync, we pack related files to apply at once. (Avoiding a mixture of unmatched versions of manifest, style.css and main.js).
Therefore customisation sync might be too expensive for files for which we can use the each of newest versions.
Since v0.19.20, we can use hidden file sync without complicated settings, for files which have not been handled in customisation sync.
If this is convenient for you, please give it a try!

P.S. [Screwdriver](https://github.com/vrtmrz/obsidian-screwdriver) also helps us with these extra files.

**Comment by dosoft** [comment]
> P.S. [Screwdriver](https://github.com/vrtmrz/obsidian-screwdriver) also helps us with these extra files.

The Screwdriver works for me. Using hidden file sync is a bit unpredictable. Actually, I can't anticipate how all the settings work, so I opt for a manual and straightforward way to sync. Thanks!
